docs/3.0.0-beta.x/getting-started/troubleshooting.md

```diff
@@ -50,7 +50,7 @@ When this new plugin release, there will be two versions:
 - Community Edition
 - Enterprise Edition
 
-By default, the Community Edition will include 3 administrators and 3 pre-defined roles (Administrators, Editor, Author). Upgrading to the Enterprise Edition will unlock an unlimited number of administrators and roles.
+By default, the Community Edition (CE) will be limited to 3 pre-defined roles (Administrators, Author). Upgrading to the Enterprise Edition (EE) will allow customers to create an unlimited number of customizable roles.
```

Comment:
`By default, the Community Edition will be limited to 3 pre-defined roles (Super Admin, Editor & Author). The roles can be customized to restrict the permissions' scope on specific collections or single types.`